generation/new_client_hermetic_build/README.md

```diff
@@ -0,0 +1,160 @@
+* `--destination-name`
+
+The corresponding value in the Cloud Drop page is `api_short_name`.
+
+Example: `alloydb`
+
+> [!IMPORTANT]
+> `api_short_name` is not always unique across client libraries.
+> In the instance that the `api_short_name` is already in use by an existing client library, you will need to determine a unique name.
+> See example under [Advanced Options](#Example with duplicate api_short_name).
+
+### Proto path (`proto_path`)
+
+The script takes "proto path" parameter. This is the path from the internal `google3/third_party/googleapis/stable` root to the
+directory that contains versions (e.g., "v1" or "v2").
+Note that the internal `google3/third_party/googleapis/stable` directory is mirrored externally in https://github.com/googleapis/googleapis/blob/master/.
+
+For example, if the buganizer ticket includes:
+
+> Link to protos: `http://...(omit).../google/cloud/alloydb/v1alpha/alloydb_v1alpha.yaml`.
+
+then the corresponding external mirrored proto is here: https://github.com/googleapis/googleapis/blob/master/google/cloud/alloydb/v1alpha/alloydb_v1alpha.yaml.
+
+Therefore, the "proto path" value we supply to the command is `google/cloud/alloydb`.
+
+We will publish a single module for a service that includes all versions in this path. Once the service has been published once, any future additional versions will automatically be generated via OwlBot.
+
+### Name pretty (`name_pretty`)
+
+The corresponding value in the Cloud Drop page is `title`.
+
+Example: `AlloyDB API`
+
+### Product Docs (`product_docs`)
+
+The corresponding value in the Cloud Drop page is `documentation_uri`.
+The value must starts with "https://".
+
+Example: `https://cloud.google.com/alloydb/docs`
+
+### REST Docs (`rest_docs`)
+
+The corresponding value in the Cloud Drop page is `rest_reference_documentation_uri`.
+The value must starts with "https://".
+
+Example: `https://cloud.google.com/alloydb/docs/reference/rest`
+
+If they exist, add them as a flag to the python command below like:
+`--rest-docs="https://cloud.google.com/alloydb/docs/reference/rest" \`
+
+### RPC Docs (`rpc_docs`)
+
+The corresponding value in the Cloud Drop page is `proto_reference_documentation_uri`.
+The value must starts with "https://".
+
+Example: `https://cloud.google.com/speech-to-text/docs/reference/rpc`
+
+If they exist, add them as a flag to the python command below like:
+`--rpc-docs="https://cloud.google.com/speech-to-text/docs/reference/rpc" \`
+
+### API description (`api_description`)
+
+The corresponding value in the Cloud Drop page is `documentation.summary` or `documentation.overview`.
+If both of those fields are missing, take the description from the product page above. Use the first sentence to keep it concise.
+
+Example:
+``` 
+    AlloyDB for PostgreSQL is an open source-compatible database service that
+    provides a powerful option for migrating, modernizing, or building
+    commercial-grade applications.
+ ```
+
+### Transport (`transport`)
```

Comment:
No need to specify `transport` because it is parsed from BUILD.

Reply:
Sorry, part of the readme transfer again

Reply:
Done